dashboard/back-end/src/config/logger.js

```diff
@@ -0,0 +1,64 @@
+const { createLogger, format, transports, Logger } = require("winston");
+
+const config = require("./environments");
```

Comment:
Logger module to enter logs into a file. 
When we move to AWS we can move the logs to something like Loggly